10up/insert-special-characters
# Update dependency-review.yml

### Description of the Change
This follows on from https://github.com/10up/insert-special-characters/pull/157 & https://github.com/10up/.github/pull/24 to utilize an org-wide config file for the license scanning action to ensure we're utilizing GPL-compatible dependencies.

### How to test the Change
Will see if the dependency check runs correctly in the PR, iterate until it does (maybe the config file link needs to be to the raw file?).

### Changelog Entry

> Changed - license scanning to use org-wide config file.


### Credits

Props @jeffpaul.


### Checklist:
